packages/store-ui/src/ProductQuantity/NumericStepper.tsx

```diff
@@ -10,24 +11,28 @@ export interface NumericStepperProps {
 
 export const NumericStepper: FC<NumericStepperProps> = ({
   variant = 'productQuantity',
+  htmlFor = 'product-quantity',
```

Comment:
I'd suggest `id` to create a more transparent interface, since that's how we use on "real" elements

Reply:
I think `htmlFor` is what you expect to pass to a Label